scripts/fixate-requirements.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import os
+import os.path
+import sys
+from distutils.version import StrictVersion
+
+
+cwd = os.path.abspath(os.getcwd())
+FIXED_REQF = os.path.join(cwd, '../fixed-requirements.txt')
+LOCAL_REQF = os.path.join(cwd, 'requirements.txt')
+GET_PIP = "    curl https://bootstrap.pypa.io/get-pip.py | python"
+
+try:
+    import pip
+    from pip.req import parse_requirements
+except ImportError:
+    print "Download pip:\n", GET_PIP
+    sys.exit(1)
+
+
+def check_pip_version():
+    if StrictVersion(pip.__version__) < StrictVersion('7.0.0'):
+        print "Upgrade pip, your version `{}' "\
+              "is outdated:\n".format(pip.__version__), GET_PIP
+        sys.exit(1)
+
+
+def check_requiremnts_sources():
+    for f in (FIXED_REQF, LOCAL_REQF):
+        if not os.path.exists(f):
+            print "Error: file `{}' not found".format(f)
+            sys.exit(1)
+
+
+def load_requirements(file_path):
+    return tuple((r for r in parse_requirements(file_path, session=False)))
+
+
+def fixate():
+    local_reqs = load_requirements(LOCAL_REQF)
+    fixed_reqs = load_requirements(FIXED_REQF)
+
+    with open("final-requirements.txt", "w") as f:
```

Comment:
We should also use full absolute path here so it doesn't depend on the cwd where the script is called from.